modules/mono/config.py

```diff
@@ -124,6 +125,8 @@ def configure(env):
             if not mono_lib:
                 raise RuntimeError('Could not find mono library in: ' + mono_lib_path)
 
+            mono_prefix = mono_lib_path
+
```

Comment:
Why is this variable needed here?

Reply:
It's... not. gunk from a previous attempt, sorry.